index.html

```diff
@@ -220,21 +220,32 @@ <h3 class='tagline'>Drag and drop so simple it hurts</h3>
   <div class='parent' ng-controller='RepeatCtrl'>
     <label for='hy'><strong>Angular-specific example.</strong> Fancy some <code>ng-repeat</code>?</label>
     <div class='wrapper'>
-      <div class='container' dragula='"another-bag"'>
+      <div class='container' dragula='"another-bag"' dragula-model='many'>
         <div ng-repeat='text in many' ng-bind='text'></div>
       </div>
-      <div class='container' dragula='"another-bag"'></div>
+      <div class='container' dragula='"another-bag"' dragula-model='many2'>
+        <div ng-repeat='text in many2' ng-bind='text'></div>        
+      </div>
+    </div>
+    <div class='wrapper'>
+      <div class='container'><pre>{{many | json}}</pre></div>
+      <div class='container'><pre>{{many2 | json}}</pre></div>
     </div>
     <pre>
       <code>
-&lt;div class=&#039;container&#039; dragula=&#039;&quot;another-bag&quot;&#039;&gt;
-  &lt;div ng-repeat=&#039;text in many&#039; ng-bind=&#039;text&#039;&gt;&lt;/div&gt;
-&lt;/div&gt;
-&lt;div class=&#039;container&#039; dragula=&#039;&quot;another-bag&quot;&#039;&gt;&lt;/div&gt;
+&lt;div class='wrapper'&gt;
+      &lt;div class='container' dragula='&quot;another-bag&quot;' dragula-model='many'&gt;
+        &lt;div ng-repeat='text in many' ng-bind='text'&gt;&lt;/div&gt;
+      &lt;/div&gt;
+      &lt;div class='container' dragula='&quot;another-bag&quot;' dragula-model='many2'&gt;
+        &lt;div ng-repeat='text in many2' ng-bind='text'&gt;&lt;/div&gt;        
+      &lt;/div&gt;
+    &lt;/div&gt;
 
-app.controller('RepeatCtrl', ['$scope',
-  function ($scope) {
+app.controller('RepeatCtrl', ['$scope', 'dragulaService',
```

Comment:
Yes, I forgot to remove it... why jshint didn't tell anything? It doesn't check example.js?